### ISSUE TITLE
Alert dialog doesn't get keyboard focus upon appearing

### ISSUE THREAD

**Issue by mkruselj** [surge-synthesizer/surge#7495]
Steps to reproduce:

1. Enable focus debugger
2. Load a patch, any patch
3. Modify it a little bit
4. Click Save Patch button
5. Click OK

What you will get is the Overwrite Patch prompt, which is good. What is not good is that this dialog doesn't have keyboard focus, so you cannot just press Enter to OK or Esc to Cancel. Would be nice to have this fixed :)

**Comment by baconpaul** [comment]
Ahh this is a great example of 'the problem you said is the problem is not the problem'

The alert box *does* get focus. If you look you see the overall screen has focus then tab takes you to message then ok then cancel

The problem is that Enter and Escape go to the alert box and we don't handle them.

It's not a focus thing. It's a don't ignore those keys thing.

Think I can fix that with a key handler in the alert box pretty easily. I'll do so.

**Comment by mkruselj** [comment]
I thought the solution for this would be to indeed focus the OK button, rather than just respond to enter and escape keys (altho that is fine too). Because this also impacts the screen reader situation. does it not?

**Comment by baconpaul** [comment]
the screen reader flow is correct. it announces the title since that has focus then you tab to the message ok and cancel

if you focus ok, escape won't cancel
